rust/pact_models/src/message.rs

```diff
@@ -195,9 +195,9 @@ impl Message {
                   description,
                   provider_states,
                   contents: body_from_json(json, "contents", &None),
-                  matching_rules: matchers_from_json(json, &None),
+                  matching_rules: matchers_from_json(json, &None)?,
                   metadata,
-                  generators: Generators::default()
+                  generators: Generators::default(),
```

Comment:
Comma needed?

Reply:
It's just a style preference. Rust allows (and some Rust style guides encourage) using trailing commas for multi-line situations like this.
My reasons for doing it are:
- It will make diffs shorter when people later need to add/remove items from the end of the list.
- It makes all lines have the same exact syntax, which I prefer aesthetically.

I only added trailing commas during this PR where I was already making changes nearby.